src/site/content/en/blog/sanitizer/index.md

```diff
@@ -0,0 +1,380 @@
+
+(A detailed explanation is in the [Explainer](https://github.com/otherdaniel/purification/blob/strings-explainer/explainer-strings.md))
+
+The [Sanitizer API](https://wicg.github.io/sanitizer-api/) has learned from the DOMPurify experience and is designed to eliminate the need for double parsing and to clarify the parsing context as follows.
+
+
+## Current status
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Step</th>
+        <th>Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>1. Create explainer</td>
+        <td><a href="https://github.com/WICG/sanitizer-api/blob/main/README.md">Complete</a></td>
+      </tr>
+      <tr>
+        <td>2. Create draft of specification</td>
+        <td><a href="https://wicg.github.io/sanitizer-api/">Complete</a></td>
+      </tr>
+      <tr>
+        <td>3. Gather feedback and iterate on design</td>
+        <td><a href="https://docs.google.com/document/d/1kBpk-dhCukPphfH_8vkOgdUDcM9bqVuhsK0pRZzwapM/edit?resourcekey=0-4tFsd4s6aLbvvr2ys_9F4A#Feedback">In progress</a></td>
+      </tr>
+      <tr>
+        <td>4. Origin trial</td>
+        <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/OrWQnXVQJ0A/m/TbF-0Dw3BQAJ">Ready for Trial</a></td>
+      </tr>
+      <tr>
+        <td>5. Launch</td>
+        <td>Not started</td>
+      </tr>
+    </tbody>
+    <caption>Current Status</caption>
+  </table>
+</div>
+
+
+## How to enable the Sanitizer API
+
+
+### Enabling via about://flags or CLI option
+
+
+#### Chrome
+
+Chrome is in the process of implementing the Sanitizer API, and when Chrome 93 is released, you can try out the behavior by enabling `about://flags/#enable-experimental-web-platform-features`. In earlier versions of the Canary/Dev channel, you can enable it via `--enable-blink-features=SanitizerAPI` and try it out for yourself right now. More details about using a flag can be found [here](https://www.chromium.org/developers/how-tos/run-chromium-with-flags).
+
+
+#### Firefox
+
+Firefox also implements the Sanitizer API. You can set the `dom.security.sanitizer.enabled` flag to `true` in `about:config` and it works as experimental.
+
+### Feature detection
+
+```js
+if (window.Sanitizer) {
+  // Sanitizer API is enabled
+}
+```
+
+### Feedback
+
+If you try this API and have some feedback, we'd love to hear it. GitHub issue is a best place to share your thoughts and discuss with folks interested in this API and spec authors.
+
+- <https://github.com/WICG/sanitizer-api/issues>
+
+If you find any bugs or unexpected behavior in Chrome's implementation, file a bug to report it. Select the `Blink>SecurityFeature>SanitizerAPI` components and share details to help implementers track the problem.
+
+- <https://new.crbug.com/>
+
+
+### Demo
+
+To see the Sanitizer API in action check out the Sanitizer API Playground by [Mike West](https://twitter.com/mikewest):
+
+- <https://sanitizer-api.dev/>
+
+
+### Browser compatibility
```

Comment:
I think browser compat should be mentioned much earlier in the post

Reply:
I feel it's ok to be here.

Reply:
I read through the post one more time, and still think this out of place here. There's already a mention of how to enable this in Firefox before this section that explains Mozilla's stance. 

My suggestion is to move this up and merge with the "Current status" section.

Reply:
move to that section.